disable-breaking-updates.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+"""
+Disable breaking updates which will prompt users to download a deb or tar file
+and lock them out of Discord making the program unusable.
+
+This will dramatically improve the experience :
+
+ 1) The maintainer doesn't need to be worried at all times of an update which will break Discord.
+ 2) People will not be locked out of the program while the maintainer runs to update it.
+
+"""
+
+import json
+import os
+from pathlib import Path
+
+XDG_CONFIG_HOME = os.environ.get("XDG_CONFIG_HOME")
+if Path(f"{XDG_CONFIG_HOME}/discord/settings.json").is_file():
+    with open(f"{XDG_CONFIG_HOME}/discord/settings.json", "r+") as settings_file:
```

Comment:
A `Path()` object has `.open()` method, no need to repeat the path. You can use it like so:
```suggestion
settings_path = Path(f"{XDG_CONFIG_HOME}/discord/settings.json")
if settings_path.is_file():
    with settings_path.open("r+") as settings_file:
```